setup/src/main/java/io/jenkins/jenkinsfile/runner/JenkinsfileRunnerRule.java

```diff
@@ -46,7 +54,16 @@ protected ServletContext createWebServer() throws Exception {
         server = new Server(queuedThreadPool);
 
         WebAppContext context = new WebAppContext(warDir.getPath(), contextPath);
-        context.setClassLoader(getClass().getClassLoader());
+        String javaVersion = System.getProperty("java.version");
+
+//        if (javaVersion.startsWith("1.")) {
```

Comment:
Let's remove the commented out code?

Also, if you mean to keep it, then I guess you'd use `isPostJava8()`